src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -133,6 +133,28 @@ public function accountWasDeleted($email)
     }
 
     /**
+     * @Given his account was deleted
+     */
+    public function hisAccountWasDeleted()
+    {
+        $user = $this->sharedStorage->get('user');
+
+        $this->userRepository->remove($user);
+    }
+
+    /**
+     * @Given there is an administrator identified by :email
+     */
+    public function theStoreHasCustomerServiceAccountIdentifiedBy($email)
```

Comment:
Please change customer service to administrator in shared storage and method name too.